neps/nep-0360.md

```diff
@@ -0,0 +1,70 @@
+
+## Motivation
+### NEAR is well below capacity
+NEAR had ~640k transactions in the last 24 hours according to the [block explorer](https://explorer.near.org/). This is probably a slight overestimate since it includes failed transactions from Aurora, the most active contract on the network. See the contract [transaction history](https://explorer.near.org/accounts/aurora) to see a sample of recent failures.
+
+Let’s be generous and say the network averages 8 successful transactions per second on average. An FT transfer is 8 tgas, while a swap on Ref Finance (with an existing exchange balance) is 14 tgas. The gas limit per block is 1000 Tgas [source](https://github.com/near/nearcore/blob/f951d6ed8e491ef842da6bd9448d3242472c126b/nearcore/res/mainnet_genesis.json#L15). Suppose the average smart contract transaction uses 10 tgas. This implies ~100 transactions per block per shard, assuming no other bottlenecks. With 4 shards on mainnet that means the network is operating at approximately 2% capacity.
+
+NEAR is operating nowhere close to capacity and currently has a fraction of the activity of the major L1s - it has the highest mcap/TVL ratio of any chain in the top 30 on [Defi Llama](https://defillama.com/chains). At $10, a fungible token transaction is 0.8 cents. On Solana, a network with at least 100x the current transaction volumes of NEAR, an SPL token transfer is 0.000005 SOL (~0.02 cents, 97% cheaper).
+
+## Rationale and alternatives
+### Prior Art
+The last major discussion around NEAR's gas prices was in 2020 [NEP-92](https://github.com/near/NEPs/issues/92). The original intent was to leave room to reduce fees later. The increase from NEP-92 was done shortly after the mainnet launch, and NEAR was 90%+ cheaper at the time.
+
+### Benefits and Ecosystem Impact
+NEAR is a volatile asset and the price increases faster than network activity. Increased gas costs when there is no shortage of block space is effectively a tax paid by users due to speculative market activity.
+
+While paying a cent for an average transaction might not seem like an issue for an infrequent user, these costs add up for liquidiy providers/arbitrageurs who improve the efficiency of markets on-chain. This makes NEAR look even worse relatively if there is another rally in the token price without corresponding network growth.
+
+Cutting fees will make NEAR even more competitive with top L1s. The ecosystem has had considerable momentum in the last few months, and NEAR’s sharding model will continue to generate interest among developers. A reduction in fees will enable more applications to build on NEAR in an economically viable way, which will drive consumer adoption.
+
+## Specification
+This proposal is to reduce the minimum price of gas from 0.0001 N/Tgas to 0.00001 N/Tgas. The total cost of a transaction is gas price * gas amount.
```

Comment:
What is the motivation for this particular price? Just returning to the 90% lower fee?

Reply:
Moreover, something that is not mentioned is why not reduce the minimum gas price by (say) 100x. [This comment](https://gov.near.org/t/proposal-reduce-the-gas-floor-price-on-the-near-network/20980/14) on the forum is answering the question, but I think we need to provide a more concise reason.

Some thoughts here: What will happen today (with no dynamic resharding) if the minimum gas price is `1yoctoNEAR/Gas`? I guess the gas price will be much lower than today and more unstable. (How unstable?)

What will happen when dynamic resharding is added? There should be a clear strategy about how to create new shards based on network activity/capacity, which will be directly related to the current gas price. One simple approach is to add/remove shards based on an upper/lower threshold disconnected from the minimum gas price. Minimum gas price will be relevant mostly when there is a single shard due to low activity.

Reply:
How are full blocks handled currently? [The documentation](https://docs.near.org/concepts/basics/transactions/gas#gas-price) mentions that gas prices can increase per block but only by 1%. Without dynamic resharding there needs to be more flexibility in the fee market.

> One simple approach is to add/remove shards based on an upper/lower threshold disconnected from the minimum gas price. Minimum gas price will be relevant mostly when there is a single shard due to low activity.

This is a reasonable approach. If the market-clearing gas price exceeds some multiple of the floor for more than a specified number of blocks it makes sense to re-shard. What level of gas price volatility is acceptable depends on the performance implications of resharding. I'm not familiar enough with the proposed implementation to make a judgement there.